packages/client/src/comp/Initialize.tsx

```diff
@@ -4,12 +4,14 @@ import { useAppHistory } from '../lib/path'
 import { InitialDataContainer, VoterContainer, useDeepMemoize } from '../lib/unstated'
 import { initializeAnalytics } from '../lib/analytics'
 import { UTM } from '../common'
+import { useLanguageSuggestion } from '../lib/languages'
```

Comment:
I don't think `useLanguageSuggestion` is a hook right?  If not, let's call it `suggestLanguageToast` or something like that.